docs/DB_API.md

```diff
@@ -838,3 +860,23 @@ Returns:
     * An empty object `{}`
   * Rejects with:
     * Any error from the underlying storage system (wrapped in `error.wrap()`)
+
+## updateTotpToken(uid, tokenData)
+
+  Update the TOTP token for the user.
+
+  Parameters:
+
+  * `uid` (Buffer16):
+    The uid of the owning account
+  * `tokenData`
+    * `verified`: Boolean whether TOTP token has been verified
+    * `enable`: Boolean whether TOTP token is enabled
```

Comment:
This only occurred to me while reviewing the auth server PR. Should it be `enabled` rather than `enable`? That way sounds more like a boolean state and less like an instruction to me.

Reply:
Yikes, looks I am not consistent with these fields/docs. I will update to use `enabled`.